lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -226,5 +229,5 @@ def _prefetch_scores(self, course_structure, course):
                 course.id, self.student.id, scorable_locations
             )
             self._submissions_scores = submissions_api.get_scores(
-                unicode(course.id), anonymous_id_for_user(self.student, course.id)
+                unicode(course.id), self.student.id
```

Comment:
This also saves us query counts, but I'm not 100% convinced it's an entirely safe change to make. Still investigating.